feat(kmp): update getting started and add manual setup page (#11497)
getsentry/sentry-docs: platform-includes/getting-started-install/kotlin-multiplatform.mdx

```kotlin {filename:shared/build.gradle.kts}
plugins {
  id("io.sentry.kotlin.multiplatform.gradle") version "{{@inject packages.version('sentry.kotlin.kmp', '0.9.0') }}"
}
```

> The plugin installs the Sentry KMP dependency and if you are using the Cocoapods plugin it will also install the Sentry Cocoa dependency.

Reviewer: ```suggestion
The plugin installs the Sentry KMP dependency. If you're using the Cocoapods plugin, it will also install the Sentry Cocoa dependency.
```
